SocketRocket/SRWebSocket.m

```diff
@@ -1383,6 +1383,16 @@ - (void)_sendFrameWithOpcode:(SROpCode)opcode data:(id)data;
     
     [self _writeData:frame];
 }
+static BOOL evaluateServerTrust(SecTrustRef serverTrust) {
+    BOOL isValid = NO;
+    SecTrustResultType result;
+
+    OSStatus status = SecTrustEvaluate(serverTrust, &result);
+    
+    isValid = ((result == kSecTrustResultUnspecified || result == kSecTrustResultProceed) && status == 0);
```

Comment:
If you are pinning, you may actually only want to consider kSecTrustResultUnspecfied valid.

SecTrustResultProceed is defined as "The user indicated that you may trust the certificate for the purposes designated in the specified policies. ". FWIW, though, these constants are confusingly named by Apple.

See: https://developer.apple.com/library/mac/documentation/Security/Reference/certifkeytrustservices/index.html#//apple_ref/c/tdef/SecTrustResultType